components/header/Buttons/Language.tsx

```diff
@@ -0,0 +1,35 @@
+import Icon from "deco-sites/bolovo-store/components/ui/Icon.tsx";
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import { usePartialSection } from "deco/hooks/usePartialSection.ts";
+
+export default function Lenguage({ countryFlag }: {
```

Comment:
Erro de escrita aqui, prefira ser mais especifico como `LanguageSwitcher`
```suggestion
export default function Language({ countryFlag }: {
```